### ISSUE TITLE
toggling coffin drawer view on desktop/laptop

### ISSUE THREAD

**Issue by tim-peterson** [fat/coffin#2]
On the demo, would be nice to restore that clickable button to toggle the display of the coffin drawers. Without it, it is hard for people on desktops/laptops to see how `coffin` works.

**Comment by heyvito** [comment]
Agreed. It's impossible to open the coffing drawers using a computer and resizing the browser.

**Comment by codekipple** [comment]
Also i'm having a go implementing coffin where i provide the typical menu button with the three stripes. The idea this toggles coffin incase people don't know they can slide.

At the moment putting data-coffin="touch" will only close the coffin drawer.

**Comment by marco-faustinelli** [comment]
Demo? What demo?
